dandischema/models.py

```diff
@@ -665,60 +688,76 @@ class PublishActivity(Activity):
 
 class Locus(DandiBaseModel):
     identifier: Union[Identifier, List[Identifier]] = Field(
-        description="Identifier for genotyping locus", nskey="schema"
+        description="Identifier for genotyping locus.", nskey="schema"
     )
     locusType: Optional[str] = Field(None)
     _ldmeta = {"nskey": "dandi"}
 
 
 class Allele(DandiBaseModel):
     identifier: Union[Identifier, List[Identifier]] = Field(
-        description="Identifier for genotyping allele", nskey="schema"
+        description="Identifier for genotyping allele.", nskey="schema"
     )
     alleleSymbol: Optional[str] = Field(None)
     alleleType: Optional[str] = Field(None)
     _ldmeta = {"nskey": "dandi"}
 
 
 class GenotypeInfo(DandiBaseModel):
-    locus: Locus = Field(description="Locus at which information was extracted")
-    alleles: List[Allele] = Field(description="Information about alleles at the locus")
-    wasGeneratedBy: Optional[List["Session"]] = Field(None, nskey="prov")
+    locus: Locus = Field(description="Locus at which information was extracted.")
+    alleles: List[Allele] = Field(description="Information about alleles at the locus.")
+    wasGeneratedBy: Optional[List["Session"]] = Field(
+        None,
+        description="Information about session activity used to determine genotype.",
+        nskey="prov",
+    )
     _ldmeta = {"nskey": "dandi"}
 
 
 class RelatedParticipant(DandiBaseModel):
     identifier: Optional[Identifier] = Field(None, nskey="schema")
-    name: Optional[str] = Field(None, title="Name of the participant", nskey="schema")
+    name: Optional[str] = Field(
+        None, title="Name of the participant or subject", nskey="schema"
+    )
     url: Optional[HttpUrl] = Field(
-        None, title="URL of the related participant", nskey="schema"
+        None, title="URL of the related participant or subject", nskey="schema"
     )
     relation: ParticipantRelationType = Field(
-        title="Participant relation",
-        description="Indicates how the current participant is related to the other participant "
-        "This relation should satisfy: Participant <relation> relatedParticipant",
+        title="Participant or subject relation",
+        description="Indicates how the current participant or subject is related"
+        " to the other participant or subject  This relation should "
+        "satisfy: Participant/Subject <relation> relatedParticipant/Subject",
```

Comment:
```suggestion
        description="Indicates how the current participant or subject is related "
        "to the other participant or subject. This relation should "
        "satisfy: Participant/Subject <relation> relatedParticipant/Subject.",
```
for consistency with the rest